src/components/PopupRevisionOverview/PopupRevisionOverview.js

```diff
@@ -96,13 +104,14 @@ function getSelectOptions(revisionObjects, leftSelect, rightSelect) {
 }
 
 /**
+ * Component that renders the PopupRevisionOverview component, which displays an Revisieoverzicht with a list of beleidskeuzes, which the user can compare two beleidskeuzes of.
+ *
+ * @Component
  *
- * @param {object} props
  * @param {boolean} revisionOverviewOpen - Indicating if the revision overview is open
- * @param {function} setRevisionOverviewOpen -
- * @param {object} dataObject - Contains the object of the detail page we are viewing
- * @param {array} revisionObjects - Array containing revisions of the object
- * @returns
+ * @param {function} setRevisionOverviewOpen - Function to open/close the RevisionOverviewOpen popup.
+ * @param {object} dataObject - Parameter containing the collection of information of a beleidskeuze.
```

Comment:
> Parameter containing the collection of information of a beleidskeuze.
I think 'the collection of information' is a bit vague and can be more specific, especially combined with 'a beleidskeuze'. Specifically it contains the data of the object the user is viewing on the detail page.

Reply:
Agreed, I've changed the parameter description.